app/assets/javascripts/audits/batch-allocation.js

```diff
@@ -0,0 +1,38 @@
+window.GOVUKAdmin = window.GOVUKAdmin || {};
+window.GOVUKAdmin.Modules = window.GOVUKAdmin.Modules || {};
+
+(function (Modules) {
+  "use strict";
+
+  Modules.BatchAllocation = function () {
+    this.start = function ($root) {
+      this._initElements($root);
```

Comment:
What do underscores in variable names mean in this context? I know they're traditionally meant to indicate `private`, but surely there's no need to do this with the GOV.UK module setup, because `.start` is the only function that's ever called externally?

Reply:
I find defining public/private methods very useful, as in the module the only public method, by contract is `start`, hence is the only one that we need to test. By following the convention os using `_` for private methods we make explicit that we can change them without breaking anything, and that we don't need to explicitly test them.

I am addressing your changes so we get this feature out, and ask in the frontend channel for current conventions around using `_` for private methods.

Reply:
> in the module the only public method, by contract is `start`

Yes, that's what I meant - by definition everything that's not `start` is inferred to be `private`. I was just confused as to why you would mark some methods `private`, but not others (also it's kind of pointless to mark them, given that we have this convention).